forge/src/runner.rs

```diff
@@ -13,7 +13,31 @@ use foundry_evm::{
 use proptest::test_runner::TestRunner;
 use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
 use serde::{Deserialize, Serialize};
-use std::{collections::BTreeMap, fmt, time::Instant};
+use std::{
+    collections::BTreeMap,
+    fmt,
+    time::{Duration, Instant},
+};
+
```

Comment:
please some basic docs, that this represents a set of tests and how long they took